lib/runner.js

```diff
@@ -198,3 +198,5 @@ Runner.prototype.run = function (options) {
 
 	return Promise.resolve(this.tests.build(this._bail).run()).then(this._buildStats);
 };
+
+Runner._chainableMethods = chainableMethods.chainableMethods;
```

Comment:
Should we wrap these options in a generator function? So we aren't exporting a mutable copy?